packages/manager/apps/key-management-service/src/hooks/useKmsGuides.ts

```diff
@@ -0,0 +1,42 @@
+import { useEnvironment } from '@ovh-ux/manager-react-shell-client';
+import { GuideItem } from '@ovhcloud/manager-components';
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+type GuideLinks = Record<string, GuideItem[]>;
+
+export function useKmsGuides() {
+  const { ovhSubsidiary } = useEnvironment().getUser();
+  const { t } = useTranslation('key-management-service/guide');
+  const [kmsGuides, setKmsGuides] = useState<GuideItem[]>([]);
+
+  const KMS_GUIDES: GuideLinks = {
+    FR: [
+      {
+        id: 1,
+        href:
+          'https://help.ovhcloud.com/csm/fr-documentation-manage-operate?id=kb_browse_cat&kb_id=3d4a8129a884a950f07829d7d5c75243',
```

Comment:
Maybe we should put thoses links into a const file ?

Reply:
I'll do it when we have the final links